client/landing/gutenboarding/hooks/use-on-site-creation.ts

```diff
@@ -85,11 +91,14 @@ export default function useOnSiteCreation(): void {
 
 				const go = async () => {
 					if ( planProduct || domainProduct ) {
-						const cart: ResponseCart = await wpcom.getCart( newSite.blogid );
-						await wpcom.setCart( newSite.blogid, {
-							...cart,
-							products: [ ...cart.products, planProduct, domainProduct ].filter( Boolean ),
-						} );
+						const cart: ResponseCart = await wpcomGetCart( String( newSite.blogid ) );
```

Comment:
Since `wpcomGetCart` only accepts strings, we have to convert `newSite.blogid` to a `string` and I don't think it's necessary since template strings (use to build the GET url) can handle numbers as well. 
I looked at the types, and `blogId` is handled as a number everywhere, so I think we can safely leave the conversion out.